src/System.Net.NameResolution/tests/FunctionalTests/GetHostEntryTest.cs

```diff
@@ -23,18 +23,15 @@ public class GetHostEntryTest
         [Theory]
         [InlineData("")]
         [InlineData(TestSettings.LocalHost)]
-        public Task Dns_GetHostEntry_HostString_Ok(string hostName) 
-            => TestGetHostEntryAsync(() => Task.FromResult(Dns.GetHostEntry(hostName)));
+        public Task Dns_GetHostEntry_HostString_Ok(string hostName) => TestGetHostEntryAsync(() => Task.FromResult(Dns.GetHostEntry(hostName)));
```

Comment:
This is fine, but FWIW what I meant in my previous comment was that just the "=>" should be on the previous line, e.g.
```C#
public Task Dns_GetHostEntry_HostString_Ok(string hostName)  =>
    TestGetHostEntryAsync(() => Task.FromResult(Dns.GetHostEntry(hostName)));
```
instead of:
```C#
public Task Dns_GetHostEntry_HostString_Ok(string hostName) 
    => TestGetHostEntryAsync(() => Task.FromResult(Dns.GetHostEntry(hostName)));
```
but all on the same line is fine, too, as long as it's not terribly long.